pages/tasks/team_details.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.base import Base
+from pages.tasks.regions.task import Task
+
+
+class TeamDetailsPage(Base):
+
+    _name_locator = (By.CSS_SELECTOR, '.content-container > h2')
+    _task_list_header_locator = (By.CSS_SELECTOR, 'div.task-list-container > h3')
+    _available_tasks_list_locator = (By.CSS_SELECTOR, '.task-list > li')
+
+    @property
+    def _page_title(self):
+        return '%s | Mozilla One and Done' % self.name
+
+    @property
+    def name(self):
+        # need to remove ' Team' from the element
+        return self.selenium.find_element(*self._name_locator).text[:-5]
```

Comment:
I was thinking you'd use the same element, i.e., the `h2` in this case. But rather than try to return just the name you'd return all the text inside the `h2`, which would represent the page heading, hence the suggested name change. Then in the test you would check for a contains rather than an equals.